File: LNPG_01_TIPOS_E_VARIAVEIS/C/celsius_para_fahrenheit_4.c
#include <stdio.h>

int main() {
    float temperature_celsius_number_user;
    printf("Insira um numero (em celsius) e veja o equivalente a Fahrenheit.\n°C ");
    scanf("%f", &temperature_celsius_number_user);
    
    float current_temperature_user_on_fahrenheit;
    current_temperature_user_on_fahrenheit = (temperature_celsius_number_user * 1.8) + 32;
    
    printf("A temperatura inserida (°C %.2f) e equivalente a °F %.2f em Fahrenheit.", temperature_celsius_number_user, current_temperature_user_on_fahrenheit);
 
    return 0;
}

File: LNPG_01_TIPOS_E_VARIAVEIS/Haskell/celsius_para_fahrenheit_4.hs
import Text.Printf

main :: IO ()
main = do
    putStrLn "Insira uma temperatura em °C, a qual sera transformada em °F."
    temperature_celsius_number_user <- readLn :: IO Float
    
    let current_temperature_user_on_fahrenheit = ((temperature_celsius_number_user * 1.8) + 32)
    
    printf "A temperatura inserida (°C %.2f) e equivalente a °F %.2f em Fahrenheit." temperature_celsius_number_user current_temperature_user_on_fahrenheit
